### DISCUSSION TITLE
申请个激活码

### DISCUSSION THREAD

**Discussion by HuiLiang1994** [LFTEC/LearningFucker#48] [General]
qlTd6CbPoOOPn/p1FNEd3H1BQP1+6PDfTRKjjVAm/X0A3w3UD/TaDGLhOWVBskBGVYh
AOpP5/oNiwlovD8gWbr1IPuJSnGH8+UkXkUO4dsu2WLxI1PerekS7gbpjwiMHBtcCDE
7Z/MplHSYPdkY7MTusbs/tKgMeSO6JdNXfOy4b3mz5RxmzrkyZdfUNINt8cpsqmhqb0
IEegWtPeP49r2VMxUkEfY58EU1FxWzANkdHeummJoq/tHbjzHNeaFODGcMCWJwDYrOS
xI+ePpc2XsQkC5BN7ZneXgcDauCLqaXMGkfYi3NSoabEc+X/7Iek1A3/lJgA/FN1oW2
ofXqi8DJzuAXAngZQzF8JyNLdbH/VhMk/LdaBvxv7ya0d2fcvJV5MzJ+AgrodhedLd0
WJB12734sbWClFgmWgCUIb0vg/2zkVQUqEd/mJmYoaPM9fDeBbDqX+Tuf+QHK8YBaoB
UZZzKuWYQQhtnC+HbwsoHrUI85A9D8ga3bs8XeaYVhlBIVRvRWLgUMgULf0lQ7AuLXl
bsg/v2sIvraTEiVKBLfjz/Nw/egCOGAGkwXCBi/3xK1jFg/8e9CzZgr2HfTcSYgjgtK
S5P+X+EOV+3ncUAqTsio4pvYfvlMKJ57y361AiX9dZxeSof9UFtNSQSvRkTqpS7OyqQ
D/Qh4N+7mFQBg+/7o0HbiB5WIY5wzCa4ci8j1QL9HwitKwJMjCGGc8hPx7F1PgV3rgB
fwhKH9Qdoj/1sv/Vefg18kE/Pujog9Ehyotg4ccQLfWbJUTHRK495j7Q5DqOW8lBqDD
YPRa3M4YidRJ9iQMgRBEZRFjknDJXDkdLjx9Rr0XyeJXHj97lkvhtfGvHYncp+4Ud5X
0toJnB1dbfqqR0r4FBWYG2eKrlFZpzDXv7zw9XxSAu2FBH/i5ZfwSNPibeG8ba6dNTP
UGEzZZdmSyzpmDliFBFFQm3VTBqAThCreuEMv7JJhgOPwXtaFBlnablvcpb4YG0R4ci
bxJxdKLTlASlrFNE+EKRegDb5blLg1tiK2l/SSy3K0DiNdlmJ/tJU64EGruwmRBxUtf
Z991w+ispBaxjcV1pBjYHrEpcfdfPBFZgQktkZ7f1at5lLn2Www2vMUIz8qeS1HZruc
b+zdlGtd96DfMztlvcH6zl5UYMOY6uglVUXgsLAnfPFav2wceZM6bwXN84DnpEjzsTz
PIa+xJpuMbd2ru0dStpOrf7FcmiwINBuBlNQjXhIuIoUGRR2klks550kJDJriUywCsc
BH5HltV9iH798DRMT7u9f6UqIpnZo8ixwKsTpUSN1lESH7L1UVOS8hFD3xmo/JMmOno
Nd1poQ1sz/hyOq8UvMgYXVGdQw==

**Comment by LFTEC** [comment]
OWNmnRnJ6nDUCXAaBhsCPAbVRYYwit8fd66YT4NXPHyVvKZhNqy8DbXZr06eMZXF9t7OyDUXKkktAa/a5nXZszJdWeSdZtampi/uI1ID72j6J4BdArjsTGfRvbu5NvPXoZ64oKqLNjPxymXq4edzF6QWh57YfIiJY+jBDon51oBgandW6OwiTnZIyGem4QaJ7dtvp/SO+Os87ucKuly9KibuhlAVZIax9LIuBRn+YoY8GRha94u1ZsxfnIFAdPL22V3x1T6udkgP3Izss6yhbxbk/w+gjz/GanBeztKACJKJMD6V3tkxjuVnp5cUVRpMxxovr2SiqiXYYIBPhwNmJHmttPZniBIFc1pNsxsn8uxaibLg9aQk6l54WCKoomPu9OUttKrWGabXvC8Pz+6SOBlNi8iWhuOMj/4VufMjcxon7r1MYY52YPZhc6RBQRUwtbMprUOMcEAN7rVzcyhRtiOHTOQeHolxd8J8EWmYg1u1mj5CNZ0CXVQtTbvxNFILSC9elBLXCVBRwV91UAtSsIq1hctW8qG5YTQf8yInaoLhLBplYju7n6FpG968V98MYLlieT2uxwXi5Ht1vVhISDKTzBAySnwT7rY0joynAWHG+sDuY5sBiIm+cCCKP5Kr//rqSWVm3g5rcRKlRFw2W3KMOaYjv7ix7rDllPTKo5QWAuMliKsC0mpwl2A85OXNodtx/dOuGqoFdTb9co1t/p/6T/Ce9q4K2yu67HUsJ21U/unK8Jt7fmwcgHk4vt9/59pCEtwVjRcSXEUY8rzLqyqXO5NodntqFIDPljBbrgQRwO8V7BDqopk6w+2HidSkgPIuds2Iig9rdoZEbH3OfwKQdi46zNIBqEdPz9On38s71Qyqso1ISxWbfL1RBSxBAd1mbCaVEwxZaJNddnGpL18RzN7vjwGNW2w2bIX7QQD9mdlzFbe+7EoD+kXDbfQ9Z1Tp14ZSLSt1CKRyyt1yFaiPfAdVkzr6HEQjsYWjid+ve1CqEGBJotS7rxNH7G+vUA86DpYYGI/gxR6TqUba1wSPPA1MrqtrxNJ40H/WJpp+HAKv2cSK5Ab94DblcqKaRyfXcdco0nY7RhQZfy+mrUbYX1AEKqNNF/14xut/XM6xOxNoM76zFssC4FaNIvRjSzP8hyljpZxgCqU3Xdqk2+Hrz1cgXb2QzEJgW8g3iiiIliIi0ysF1j7oKoZlI0i0R1VY9NGfrHDYDVm8CmQwjUpK/uKty7r9FIma+ozsGjL0glRduZiweguZ/xqhW3Fg+SYTTE+4sJv7K0cCby5bLqqXoBjeJYYmgf00MCcTLtxYbyyCgkB+AMrzEwquBDSfvnbbkpAZjlzySu/NUplYuw==
